### PR TITLE
PD-400 Azure KMS PVC Encryption Docs

### PR DESCRIPTION
**Pull request by andreipope**
PD-400 Azure KMS PVC Encryption Docs
[PVC-Enc-A4.pdf](https://github.com/portworx/pxdocs/files/4569682/PVC-Enc-A4.pdf)

### CODE REVIEW

**Comment by andreipope** [comment]
@nathan-px We must identify a good SME.

**Comment by andreipope** [comment]
@nathan-px @adityadani Just a kind reminder☺️

File: content/key-management/aws-kms/pvc-enc/_index.md [hunk 1]
```diff
@@ -1,110 +1,147 @@
...
+        requests:
+          storage: 2Gi
+
+    ```
+
+    {{<info>}}
+**NOTE:** You can use the same named secret to encrypt multiple volumes.
+    {{</info>}}
+
+    If you want to migrate encrypted volumes created through this method between two different Portworx clusters, then you must:
```
Comment (nathan-px) [inline content/key-management/aws-kms/pvc-enc/_index.md:114]: I'm a little confused by this.

File: content/key-management/gcloud-kms/pvc-enc/_index.md [hunk 2]
```diff
@@ -1,142 +1,119 @@
...
 
-### Encryption using cluster wide secret
 
-In this method a default cluster wide secret will be set for the Portworx cluster. Such a secret will be referenced by the user and Portworx as **default** secret. Any PVC request referencing the secret name as `default` will use this cluster wide secret as a passphrase to encrypt the volume.
+    ```text
+    pxctl secrets gcloud create-secret --secret_id my-unique-secret --passphrase mysecretpassphrase
+    ```
 
-#### Step 1: Set the cluster wide secret key
+    Note that Portworx uses Google Cloud KMS to encrypt your passphrase, and stores it in its internal metadata store.
```
Comment (nathan-px) [inline content/key-management/gcloud-kms/pvc-enc/_index.md:44]: maybe wrap this in an info tag and make it a note. 
```suggestion
{{<info>}}
**NOTE:** that Portworx uses Google Cloud KMS to encrypt your passphrase and stores it in its internal metadata store.
{{</info>}}
```

**Pull request merged**